urlrewriter.config

```diff
@@ -475,5 +475,5 @@
     <redirect url="/devtools/(.*)/controls/radgridview/how-to/modify-foreground-row" to="/devtools/$1/knowledge-base/kb-gridview-modify-foreground-selected-hovered-row" processing="stop" />
     <redirect url="/devtools/(.*)/controls/raddatetimepicker/styles-and-templates/styling-template-structure" to="/devtools/$1/controls/raddatetimepicker/styles-and-templates/styling-overview" processing="stop" />
     <redirect url="/devtools/(.*)/controls/radtreeview/styles-and-templates/tesmplates-structure" to="/devtools/$1/controls/radtreeview/styles-and-templates/styling-and-appearance-overview2" processing="stop" />
-
+    <redirect url="/devtools/(.*)/controls/radgridview/troubleshooting/performance" to="/devtools/$1/controls/radgridview/performance/tips-tricks" processing="stop" />
```

Comment:
The page is redirected, but it is not removed from the documentation. This means that you will be able to see it in the navigation tree. Is this intentional?